conans/test/functional/folders_access_test.py

```diff
@@ -46,6 +46,10 @@ def assert_in_local_cache(self):
             assert(self.in_local_cache == False)
    
     def source(self):
+        self.source_folder
+        self.build_folder
+        self.package_folder
```

Comment:
What is this testing? I am a bit confused that build_folder and package_folder would be defined here? I guess they are None? Please check them

Reply:
checking availability of those attributes in conanfile object.
They are None, but exist there... However I dont know if that makes sense